languages/python/README.md

```diff
@@ -51,5 +51,5 @@ These extra steps will make your track better, but are optional.
 
 ### Analyzer
 
-- [ ] Build Analyzer
+- [x] Build [Analyzer](https://github.com/exercism/python-analyzer)
 - [ ] Deploy Analyzer
```

Comment:
Isn't the Analyzer deployed too? As in, it has already commented on v2 exercises?

Reply:
Yes, I wasn't clear on whether or not "deployed" in this case meant to some v3 infrastructure, or currently in use for V2 or the research experiment.

Reply:
I understand. You can definitely consider this "deployed."

Reply:
Cool. Ticked.